bio/reference/ensembl-genomes-annotation/wrapper.py

```
@@ -0,0 +1,67 @@
+import sys
+
+from snakemake.shell import shell
+from snakemake.logging import logger
+from pathlib import Path
+
+species = snakemake.params.species.lower()
+release = int(snakemake.params.release)
+division = snakemake.params.division.lower()
+assembly = snakemake.params.assembly
+
+out_fmt = Path(snakemake.output[0]).suffixes
```

Ruff (0.14.10)

Undefined name `snakemake`

(F821)